### ISSUE TITLE
OneDrive integration/streaming

### ISSUE THREAD

**Issue by digimezzo** [digimezzo/dopamine-windows#67]
OneDrive integration/streaming would be amazing

**Comment by Riajyuu** [comment]
I think if add onedrive, there can be users requesting others  only if there are public APIs like Google drive, Dropbox, yandex, etc.

**Comment by ThePharaohArt** [comment]
Yes! please add Onedrive streaming support! This would be amazing.

**Comment by ThePharaohArt** [comment]
Do one thing at a time though begin with onedrive, see how it works then branch out to other cloud storage like GDrive etc.
